### PR TITLE
win32 color support using colorama

### PR DESCRIPTION
**Pull request by RSabet**
Added support for colors on Windows using module colorama imported only on Windows machines.

`pip install colorama`

### CODE REVIEW

**Comment by chadrik** [comment]
We should make sure that this does not error for windows users who don't have `colorama` installed.  A few options:

- add `colorama` to setup.py requirements.  cons:  gets installed for non-windows users
- try/except the import of `colorama` on windows, and print a warning informing the user to `pip install colorama` if they want colored output.  also, add something to the readme about installing colorama manually if you're on windows.

I'm in favor the first option, unless you have a better suggestion.  Can you make that adjustment, please?

**Comment by RSabet** [comment]
done option 1

**Comment by chadrik** [comment]
The reason I haven't merged this yet is because ideally we would not make `colorama` a requirement on non-windows platforms, but I'm not entirely sure how to do that. 

It appears that it's possible to express platform-specific requirements in `setup.py`:
https://stackoverflow.com/questions/6469508/is-it-possible-to-express-a-platform-specific-dependency-in-setup-py-without-bui

That said, wheels are different from eggs -- eggs build at the time of install (on a user's machine), and wheels build at the time of packaging (on the distributor's machine).  So I think to achieve the desired results I may need to build a separate wheel for each platform. 

tl;dr more experimenting is required.
